src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionsTests.cs

```diff
@@ -11,14 +11,13 @@
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 using Newtonsoft.Json.Linq;
 using Roslyn.Test.Utilities;
-using Xunit;
 using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.CodeAnalysis.LanguageServer.UnitTests.CodeActions
 {
     public class CodeActionsTests : AbstractLanguageServerProtocolTests
     {
-        [Fact]
+        [WpfFact]
```

Comment:
interesting.  why?

Reply:
This PR changes LSP code actions to require the UI thread. However, Fact doesn't seem to like switching over to the UI thread and throws an exception. I talked to DavidB and he gave the idea of using WpfFact as a workaround, which seems to fix the issue.

Reply:
Yes, WpfFact is needed when we switch to the main thread.  I guess previously it wasn't needed since we weren't using the same filtering logic in the LSP code actions (and so didn't need to jump to the UI thread).